logo.svg

```diff
@@ -0,0 +1,26 @@
+<svg
```

Comment:
Part of the starter kit for Drupal, it's effectively for making your own logos in the Drupal CMS to differentiate from the parent. Pretty useless for us, but it comes over from the Bootstrap theme starter kit to create the subtheme.